packages/grafana-ui/src/components/Slider/types.ts

```diff
@@ -0,0 +1,29 @@
+import { Orientation } from '../../types/orientation';
+
+export interface SliderProps {
+  min: number;
+  max: number;
+  orientation?: Orientation;
+  /** Set current positions of handle(s). If only 1 value supplied, only 1 handle displayed. */
+  value?: number;
+  reverse?: boolean;
+  step?: number;
+  tooltipAlwaysVisible?: boolean;
+  formatTooltipResult?: (value: number) => number;
+  onChange?: (value: number) => void;
+  onAfterChange?: (value?: number) => void;
+}
+
+export interface RangeSliderProps {
+  min: number;
+  max: number;
+  orientation?: Orientation;
+  /** Set current positions of handle(s). If only 1 value supplied, only 1 handle displayed. */
+  value?: number[];
```

Comment:
Do we have any scenario where thus could be more than 2 values? I'm thinking that we might be able to have it as `from: number, to:number`.